specification/network/resource-manager/Microsoft.Network/stable/2024-05-01/loadBalancer.json

```diff
@@ -2736,6 +2809,51 @@
       },
       "description": "Individual port mappings for inbound NAT rule created for backend pool."
     },
+    "LoadBalancerHealthPerRule": {
+      "type": "object",
+      "properties": {
+        "up": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Number of backend instances associated to the LB rule that are considered healthy."
+        },
+        "down": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Number of backend instances associated to the LB rule that are considered unhealthy."
+        },
+        "loadBalancerBackendAddresses": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LoadBalancerHealthPerRulePerBackendAddress"
+          },
+          "description": "Information about the health per rule of the backend addresses."
+        }
+      },
+      "description": "The response for a Health API."
+    },
+    "LoadBalancerHealthPerRulePerBackendAddress": {
```

Comment:
>LoadBalancerHealthPerRulePerBackendAddress [](http://example.com/codeflow?start=5&length=42)

Information about Health is not present in this definition body. 
Can the name/description of this definition be changed to reflect it ?

Reply:
So, state is the health information (ie if it's healthy or not) and reason explains why it's healthy/unhealthy. Unless I'm misunderstanding the comment, it does give some info about health.

Reply:
MUHAMMAD IHSAN ALI.

Reply:
MUHAMMAD IHSAN ALI.
![image](https://github.com/user-attachments/assets/9474d47c-aff6-4603-8f13-6a08bd44e813).